src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java

```diff
@@ -874,46 +873,37 @@ private void setRetryTimeout(long timeoutMs) {
   }
 
   private void createHiveTable() {
```

Comment:
Do we have tests covering `createHiveTable()`, `alterHiveSchema()` and `addHivePartition()`, or verified the intended behavior of the updated lambda expressions other ways?

Reply:
yes, we have multiple tests like these https://github.com/confluentinc/kafka-connect-hdfs/blob/master/src/test/java/io/confluent/connect/hdfs/avro/HiveIntegrationAvroTest.java
for each each dataformat